src/common/metrics/ratio_beds_with_covid_patients.tsx

```diff
@@ -22,24 +22,28 @@ export const RatioBedsWithCovidPatientsMetric: MetricDefinition = {
   metricNameForSummaryStat: 'Patients w/ COVID',
 };
 
+const LIMIT_LOW = 0.1;
+const LIMIT_MEDIUM = 0.15;
```

Comment:
I think this should be `LIMIT_MEDIUM = 0.14999` since High is when _patients w/ COVID_ >= 15%